world.pl

```diff
@@ -140,7 +141,8 @@
     % Keep only the previous generation of facts
     Prev is Gen - 2,
     retractall(fact(Prev,_)),
-    sleep(300),
+    config(refresh_time, RefreshTime),
```

Comment:
if we go this way, we need to implement a default value mechanism if the config is not present.